### PR TITLE
CDI-681 Spec doesn't talk about BeanConfigurator.read method

### PR DESCRIPTION
**Pull request by antoinesd**
CDI-681 Spec doesn't talk about BeanConfigurator.read method

### CODE REVIEW

**Comment by mkouba** [comment]
The spec still does not mention `BeanConfigurator.read()` at all.

**Comment by antoinesd** [comment]
branch mixup. thx

File: spec/src/main/asciidoc/core/spi.asciidoc [hunk 1]
```diff
@@ -1194,6 +1194,7 @@ CDI 2.0 introduced the `javax.enterprise.inject.spi.configurator.BeanConfigurato
 
 With `BeanConfigurator` you can perform all the operations defined in <<bean_attributes_configurator>> plus the following:
 
+* Read the bean metadata from a `BeanAttributes` or an `AnnotatedType` with one of its `read()` methods.
```
Comment (mkouba) [inline spec/src/main/asciidoc/core/spi.asciidoc]: What about https://issues.jboss.org/browse/CDI-681?focusedCommentId=13347214&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-13347214 ?

Comment (manovotn) [inline spec/src/main/asciidoc/core/spi.asciidoc]: I would opt to incorporate something indicating that the configurator will be initialized with the metadata you read:

>  `read()` methods initiate the configurator with existing `BeanAttributes` or `AnnotatedType` metadata.

**Comment by manovotn** [comment]
Looks good now

**Comment by tremes** [comment]
I think it's ok.

**Pull request merged**